packages/e2e-tests-playwright/specs/editor/various/new-post.spec.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Internal dependencies
+ */
+const { test, expect } = require( '../../../config/test' );
+
+test.describe( 'new editor state', () => {
+	test.beforeAll( async ( { testUtils } ) => {
+		await testUtils.activatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.afterAll( async ( { testUtils } ) => {
+		await testUtils.deactivatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test( 'should show the New Post page in Gutenberg', async ( {
+		page,
+		testUtils,
+	} ) => {
+		await testUtils.createNewPost();
+
+		expect( page.url() ).toEqual(
+			expect.stringContaining( 'post-new.php' )
+		);
+
+		// Should display the blank title.
+		const title = page.locator( 'role=textbox[name="Add title"i]' );
+		await expect( title ).toBeEditable();
+
+		// Trim padding non-breaking space
+		expect(
```

Comment:
`await expect(title).toHaveText('')`
does this also work for your use-case?

Reply:
TBH, I have no idea why this was written that way 😅 . But yeah, your suggestion should work?